.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,52 @@
+# Contributing
+
+The following is a set of guidelines for contributing to this project.
+These are just guidelines, not rules.
+Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+## Development Setup
+
+### The Easy Way
+
+Use the supplied [Development Container](https://containers.dev/) configuration
+to get a development environment up and running quickly.
+The configurations can be found in the `.devcontainer` directory.
+
+[GitHub Codespaces](https://github.com/features/codespaces) can provide
+a development environment in the cloud.
+GitHub Codespaces also provides a prebuilt image for this repository
+from the most recent commit on the `main` branch
+to speed up the setup process, including many of the Python runtimes
+and tools needed to run the tests.
```

Comment:
Note: This will only be true after it's been merged to `main` and enabled and Codespaces hasn't hit a spending limit.
https://docs.github.com/en/codespaces/managing-codespaces-for-your-organization/choosing-who-owns-and-pays-for-codespaces-in-your-organization